custom/icds_reports/utils/aggregation_helpers/distributed/migration_forms.py

```diff
@@ -0,0 +1,49 @@
+from dateutil.relativedelta import relativedelta
+
+from custom.icds_reports.const import AGG_MIGRATION_TABLE
+from custom.icds_reports.utils.aggregation_helpers import month_formatter
+from custom.icds_reports.utils.aggregation_helpers.distributed.base import (
+    StateBasedAggregationDistributedHelper,
+)
+
+
+class MigrationFormsAggregationDistributedHelper(StateBasedAggregationDistributedHelper):
+    helper_key = 'migration-forms'
+    ucr_date_source_id = 'static-migration_form'
+    aggregate_parent_table = AGG_MIGRATION_TABLE
+
+    def aggergation_query(self):
+        month = self.month.replace(day=1)
+        current_month_start = month_formatter(self.month)
+        next_month_start = month_formatter(self.month + relativedelta(month=1))
+
+        query_params = {
+            "month": month_formatter(month),
+            "state_id": self.state_id,
+            "current_month_start": current_month_start,
+            "next_month_start": next_month_start,
+        }
+
+        return """
+        INSERT INTO "{tablename}" (
+        	case_id, state_id, supervisor_id, month, latest_time_end_processed,
+			migration_status
```

Comment:
Where are you fetching migration_Status from? it looks its missed to be fetched in the select sub query?